src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
+# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.subsystem.subsystem import Subsystem
+
+
+class PyTest(Subsystem):
+  options_scope = 'pytest'
+
+  @classmethod
+  def register_options(cls, register):
+    super(PyTest, cls).register_options(register)
+    register('--pytest-requirements', default='pytest>=2.6,<2.7',
```

Comment:
The option scope is `pytest`, and I thought `--pytest-pytest-requirements`, `--pytest-pytest-cov-requirements` looked silly compared to `--pytest-requirements`, `--pytest-cov-requirements` etc.

That said, these should all be advanced options anyway (made them so in the review followup commit) so will be very unlikely to ever be set on the cmd line, so I can restore those prefixes. Done.